src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,70 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] storage;
+    private static final int DEFAULT_SIZE = 10;
+
+    public ArrayStack(int size) {
+        if (size < 1) {
+            throw new IllegalArgumentException("Can't create stack with size less than 1");
+        }
+
+        this.size = size;
+        storage = new int[size];
+    }
+
+    public ArrayStack() {
+        this.size = DEFAULT_SIZE;
+        storage = new int[size];
+    }
+
+    public boolean isEmpty() {
+        return top == 0;
+    }
+
+    public void push(int value) {
+        if (top == storage.length) {
+            increaseStackCapacity();
+        }
+
+        storage[top] = value;
+        top++;
+    }
+
+    public int size() {
+        return top;
+    }
+
+    public int peak () {
+        validateNotEmpty();
+
+        return storage[top - 1];
+    }
+
+    public int pop() {
+        validateNotEmpty();
+
+        int popValue = storage[top - 1];
+
+        storage[top - 1] = 0;
+        top--;
+
+        return popValue;
+    }
+
+    private void increaseStackCapacity() {
```

Comment:
Your class calls stack no need to point it second time in method name.
This is the last comment just delete word Stack from this method name and merge this to develop. You have a conflict. Find out what is "conflict" in git and resolve it